src/controllers/dlgprefcontroller.cpp

```diff
@@ -541,7 +543,9 @@ QUrl DlgPrefController::helpUrl() const {
 }
 
 void DlgPrefController::enableWizardAndIOTabs(bool enable) {
-    m_ui.btnLearningWizard->setEnabled(enable);
+    // We always enable the Wizard button if this is a MIDI controller so we can
+    // create a new mapping from scratch with 'No Mapping'
+    m_ui.btnLearningWizard->setEnabled(m_pController->isMIDI());
```

Comment:
IMO adding a `isMIDI` method to the base class is a very ugly hack. The controller class should not have any knowledge of the IO protocol. What do you think of removing the ismidi method and instead use Qt's type info?
```suggestion
    // We always enable the Wizard button if this is a MIDI controller so we can
    // create a new mapping from scratch with 'No Mapping'
    const auto* midiController = qobject_cast<MidiController>(m_pController);
    m_ui.btnLearningWizard->setEnabled(midiController != nullptr);
```
Alternative solution would be to use a virtual method, though I consider that less clean.

Reply:
Good idea, thanks!

I'll squeeze this in.
Btw forgot to set to Draft, guess I'm not quite done, yet.